.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+  - "2.7"
```

Comment:
Please replace this to 3.5, we're going to ditch the support for 2.7 in the future. Even now it is supported in a limited fashion.